packages/component/src/Styles/StyleSet/ActivityButton.ts

```diff
@@ -39,7 +39,10 @@ export default function createActivityButtonStyle() {
 
       '& .webchat__activity-button__icon': {
         height: '20px',
-        width: '20px'
+        width: '20px',
+
+        '--webchat__component-icon--color': '#707070',
```

Comment:
Should we set these via props?

Reply:
Props currently map to modifier classes mostly and you can have as many as you want. They are still somethat a static set of things. I find it perfect for UI work, when designing components from scratch. It enforces a limited set of variations.

For more complex use-cases we may want to refrain to setting the props by hand. Why would we need to opt-into the JS to set a CSS property e.g. for color??